presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveStorageFormats.java

```diff
@@ -47,6 +47,7 @@
     {
         return new StorageFormat[][] {
                 {storageFormat("ORC")},
+                {storageFormat("ORC", ImmutableMap.of("hive.orc_optimized_writer_enabled", "true", "hive.orc_optimized_writer_validate", "true"))},
```

Comment:
Thanks. Please to the similar thing for the RC file as well ;-)

Reply:
I wasn't really sure about the RCFile, the optimized RC file writer is enabled by default. Do we need an entry here to test the case that turns the optimized writer off?

Reply:
Oh, it is on by default. Then yes, it make sense to disable here. Also it make sense to enable verification when optimized writer is used.

Reply:
Got it, will address it.